extensions/ql-vscode/test/vscode-tests/no-workspace/query-history/variant-analysis-history.test.ts

```diff
@@ -72,9 +74,9 @@ describe("Variant Analyses and QueryHistoryManager", () => {
 
     disposables = new DisposableBucket();
 
-    rawQueryHistory = readJSONSync(
+    queryHistory = readJSONSync(
       join(STORAGE_DIR, "workspace-query-history.json"),
-    ).queries;
+    ).queries.map(mapQueryHistoryVariantAnalysisToDomainModel);
```

Comment:
One thing I'm not totally clear on is what exactly we're testing. So is it appropriate to use `mapQueryHistoryVariantAnalysisToDomainModel` to generate the expected data, or are we basically just calling the same function twice and comparing the result?

I also tried writing out the expected data manually and explicitly, instead of loading it from this file. The issues I ran into were:
- we duplicate a lot of data (but that's kinda the point)
- the data contains some `PLACEHOLDER`s in the file paths, because we don't know where the file will be written so we can't hard-code the path to the QL files